### ISSUE TITLE
WebSocket connection refused with 403 after update to 1.3.0

### ISSUE THREAD

**Issue by orhtej2** [YunoHost-Apps/funkwhale_ynh#229]
### Describe the bug

After upgrade to `1.3.0` I'm no longer able to play music.
When checking UI with Inspector (Firefox) I see HTTP 403 when UI tries to connect to wss://<snip-domain.tld>/api/v1/activity

UI and activity feed (latest uploads, last played tracks etc) loads just fine.

I can see my avatar and profile - I was successfully logged in.

### Context

- Hardware: VPS bought online
- YunoHost version: 11.1.20
- I have access to my server: Through SSH
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: upgrade to 1.3.0 (from 1.2.10)
- Using, or trying to install package version/branch: 1.3.0~ynh1
- If upgrading, current package version: 1.2.10->1.3.0

### Expected behavior

Funkwhale working as before, especially playing music.

### Logs

- [Upgrade log](https://paste.yunohost.org/raw/keketofigi)
- [funkwhale-service log](https://paste.yunohost.org/uzahoxarod)

**Comment by Thovi98** [comment]
Thanks for your issue!
It’s probably a nginx issue (I had to rewrite it for this version). Could you please share your Funkwhale’s ngnix logs (you may want to change your domain by a random one)?

**Comment by orhtej2** [comment]
Attached, mind you it's a mix of new and old as I uninstalled `1.3.0` and restored pre-upgrade backup of `1.2.10` (worked, yay! :))

[domain.tld-access.zip](https://github.com/YunoHost-Apps/funkwhale_ynh/files/11682104/domain.tld-access.zip)

**Comment by orhtej2** [comment]
Not sure what you mean, I've uploaded the music via normal 'Upload music from other artists' functionality.

Installing from `testing` made things worse.

I no longer see any 'dynamic' content of main page (no albums, recent activity etc), just static/empty content and red box to upper right says 'An unexpected error occurred.'

If I dig into Inspector's console I get:

```
Unexpected API error 
Object { stack: "Z@https://domain.tld/assets/index-0ab871f8.js:5:18973\na_@https://domain.tld/assets/index-0ab871f8.js:7:947\nd@https://domain.tld/assets/index-0ab871f8.js:7:4045\n", message: "Request failed with status code 502", name: "AxiosError", code: "ERR_BAD_RESPONSE", config: {…}, request: XMLHttpRequest, response: {…}, backendErrors: (605) […], isHandled: false, rawPayload: "<!DOCTYPE html>\n<html>\n<head>\n<title>502 Bad Gateway</title>\n<style>\n    body {\n        width: 35em;\n        margin: 0 auto;\n        font-family: Tahoma, Verdana, Arial, sans-serif;\n    }\n</style>\n</head>\n<body>\n<h1>502 Bad Gateway</h1>\n<p>If you see this page, your connection with the server is working but the internal service providing this path is not responding.</p>\n<p>Administrator, make sure that the service is running, and check its logs if it is not.\nThe Services page is in your webadmin, under <code>Tools > Services</code>.</p>\n<p><em>Thank you for using YunoHost.</em></p>\n</body>\n</html>\n" }
 
Array(605) [ "<", "!", "D", "O", "C", "T", "Y", "P", "E", " ", … ]
useLogger-964d3ffc.js:5:1869
```

**Comment by orhtej2** [comment]
Come to think about it let me try again, perhaps it's #206 striking again.

**Comment by Thovi98** [comment]
>  I've uploaded the music via normal 'Upload music from other artists' functionality

Yes, it’s what I meant.

Okay so if I understand well, the testing branch resolves your issue?
I will soon have a look at your issue in #206, sorry for not have looked at it yet

**Comment by orhtej2** [comment]
Yes, code from `testing` branch works for me 👍
